tidy3d/components/geometry/base.py

```diff
@@ -368,7 +368,11 @@ def _get_center(pt_min: float, pt_max: float) -> float:
     @cached_property
     def _normal_2dmaterial(self) -> Axis:
         """Get the normal to the given geometry, checking that it is a 2D geometry."""
-        raise ValidationError("'Medium2D' is not conpatible with this geometry class.")
+        raise ValidationError("'Medium2D' is not compatible with this geometry class.")
+
+    def _update_from_bounds(self, bounds: Tuple[float, float], axis: Axis) -> Geometry:
+        """Set the bounds of a geometry in the axis direction."""
+        raise ValidationError("'_update_from_bounds' is not compatible with this geometry class.")
```

Comment:
I think this should be a `NotImplementedError`

Reply:
Also, perhaps the docstring could be more clear. So this method basically makes an updated copy of the geometry that's transformed to fit within some bounds in the axis direction? from the current docstring it sound like it just sets `geometry.bounds`